chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts

```diff
@@ -56,6 +56,57 @@ export class FormValidationService {
     }
     return alphabetical;
   }
+  public isLowercase(input: string): boolean {
+    let lowercase = false;
+    const alphaChars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+    const charactersArray = input.replace(/ /g, '').split('');
+    for (const character of charactersArray) {
+      console.log(character);
+      if (alphaChars.includes(character)) {
+        lowercase = true;
+
+      } else {
+        lowercase = false;
+        console.log('character not valid');
+        break;
+      }
+    }
+    return lowercase;
+  }
+  public isUppercase(input: string): boolean {
+    let uppercase = false;
+    const alphaChars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+    const charactersArray = input.replace(/ /g, '').split('');
+    for (const character of charactersArray) {
+      console.log(character);
+      if (alphaChars.includes(character)) {
+        uppercase = true;
+
+      } else {
+        uppercase = false;
+        console.log('character not valid');
+        break;
+      }
+    }
+    return uppercase;
+  }
+  public containsSpecialChars(input: string): boolean {
+    let special = false;
+    const specialChars = '!@#$%^&*()_+=-[]{}\"|/?.>,<'.split('');
+    const charactersArray = input.replace(/ /g, '').split('');
```

Comment:
Not sure if we want to change the input by removing any spaces before we split the string.  We should be taking the input as is.  Remove the replace method and just use the split method.

Reply:
Removed replace method and left split method.